com.unity.multiplayer.mlapi/Runtime/Core/NetworkUpdateLoop.cs

```diff
@@ -373,5 +394,92 @@ private static void Initialize()
 
             PlayerLoop.SetPlayerLoop(customPlayerLoop);
         }
+
+#if UNITY_EDITOR
+        private static void UninjectSystems()
+        {
+            var customPlayerLoop = PlayerLoop.GetCurrentPlayerLoop();
+
+            for (int i = 0; i < customPlayerLoop.subSystemList.Length; i++)
+            {
+                var playerLoopSystem = customPlayerLoop.subSystemList[i];
+
+                if (playerLoopSystem.type == typeof(Initialization))
```

Comment:
nit: there is a ton of copy/paste and weird `{` scoping here. Could just extract method that takes a type for the system and a type for the network subsystem :)

Reply:
sure, I'll try to make it look fancier :)